### DISCUSSION TITLE
Multiple Plates

### DISCUSSION THREAD

**Discussion by Petervanoos** [HASwitchPlate/HASPone#54] [Q&A] [answered]
I have four plates. Is there an easy way to set up all the plates, so they are identical?

**Comment by aderusha** [comment, marked as answer]
Technically possible w/ some hacking.  If you really want to clone a setup HASPone, you can open the `automations.yaml` in your home assistant directory, copy all of the automations you deployed from blueprint for the first device, paste it several times, replace the device ID for each of the copies w/ the correct device, and then change all of the `id` numbers assigned to each automations.  The only rule on the automation ID is that they must all be unique.

**Reply by Petervanoos** [reply]
Thanks, i will give it a try.
